json-modules/json-operations/src/test/jacksonfindvalue/JacksonFindValueUnitTest.java

```diff
@@ -0,0 +1,97 @@
+package com.baeldung.jacksonfindvalue;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class JacksonFindValueUnitTest {

```

Comment:
As this is JUnit 5, we remove the `public` keyword from the test class and tests